Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups_description.md

```diff
@@ -0,0 +1,7 @@
+To allow us access to Microsoft Graph Groups, an admin has to approve our app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-groups).
+After authorizing the Demisto app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
+
+Required Permissions in the MS Graph Groups App:
+Directory.ReadWrite.All - Delegated
```

Comment:
nice to have it here, but more important to remember mentioning this in the documentation for each command what permission is required

Reply:
Will do